tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -22,7 +22,24 @@ public String name() {
   public void processFeature(SourceFeature sourceFeature, FeatureCollector features) {
     if (sourceFeature.canBeLine() && sourceFeature.hasTag("highway") &&
       !(sourceFeature.hasTag("highway", "proposed", "abandoned", "razed", "demolished", "removed", "construction"))) {
+
       String highway = sourceFeature.getString("highway");
+      String shield_text = sourceFeature.getString("ref");
+      String network_val = sourceFeature.getString("network");
+      shield_text = (shield_text == null ? null : shield_text.split(";")[0].replaceAll("\\s", ""));
+      if( shield_text != null ) {
+        if( shield_text.contains("US ") ) {
```

Comment:
This is cheap way of imputing the network without walking all the relations. The output for tile user is the same, and can be upgraded to rels later without changthe contract.